gitPractice/1-gitHomework.js

```diff
@@ -0,0 +1,45 @@
+//? practice to undrestand difference between const and let.
+//const and let are variable, on the other hand const is better than let because const variable won't be reasigned however let could be reasigned.
+// In conclusion:
+// "Var" variable can be updated and re-declared.don't need initialized.
+// "let" variable can be updated but not re-declared. don't need initialized.
+// "const" variable neither be update and re-declared. needs initialized.
+
```

Comment:
@RominaBideh  Could you explain let and const with your own words, I think I can find this description through google

Reply:
@ellietms Yes, I do. Let variable is working everywhere and also it has to have a value. we do not have to use it a lot in our code.
Const variable could update but it is specific we couldn't call it everywhere on the other hand let could be re-declared which is really not good sometimes, because it could be confusing if we use lots of let.

Reply:
@RominaBideh  "Let variable is working everywhere", where is everywhere? explain to me

"We do not have to use it a lot in our code." ==> why? (it might be a right or wrong answer)

"Const variable could update but it is specific we couldn't call it everywhere on the other hand let could be re-declared which is really not good sometimes, because it could be confusing if we use lots of let." 
 
 This explanation is not right, the way you explained const , you are saying const could update and we can not call it everywhere? what do you mean by this? ===> learn more
 
 "because it could be confusing if we use lots of let."   ==>. it is not correct